app/uk/gov/hmrc/thirdpartydeveloperfrontend/helpers/PhoneNumberHelper.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.thirdpartydeveloperfrontend.helpers
+
+object PhoneNumberHelper {
+
+  def obfuscateNumber(mobileNumber: String): String = mobileNumber.substring(0, mobileNumber.length - 4).replaceAll("\\S", "*") + mobileNumber.takeRight(4)
```

Comment:
```suggestion
  def obfuscateNumber(mobileNumber: String): String = "*" * 6 + mobileNumber.takeRight(4)
```

This looks overly complicated. Does the user need to know how many digits are in their phone number?

Reply:
On this one I was following the practice on tpd, so I assumed it was all fine
https://github.com/hmrc/third-party-developer/blob/0d448404a445f8daf930e82779ce4b797c46a4dc/app/uk/gov/hmrc/modules/mfa/utils/PhoneNumberHelper.scala#L18